src/components/clusterprovider/minikube/minikube.ts

```diff
@@ -109,6 +133,41 @@ async function stopMinikube(context: Context): Promise<void> {
     });
 }
 
+async function minikubeStatus(context: Context, notification: boolean): Promise<MinikubeInfo> {
+    if (!await checkPresent(context, notification? CheckPresentMode.Alert : CheckPresentMode.Silent)) {
+        if (notification) {
+            return;
+        }
+        throw new Error('minikube executable could not be found!');
+    }
+
+    try {
+        const result = await context.shell.exec(
+            `${context.binPath} status --format '["{{.MinikubeStatus}}","{{.ClusterStatus}}","{{.KubeconfigStatus}}"]'`);
+        if (result.stderr.length === 0) {
+            const obj = JSON.parse(result.stdout);
+            if (notification) {
+                vscode.window.showInformationMessage(`Minikube is ${obj[0]}`);
+            }
+            return {
+                running: 'Stopped' !== obj[0],
+                cluster: obj[1],
+                kubectl: obj[2],
+            } as MinikubeInfo;
+        }
+        if (notification) {
+            vscode.window.showErrorMessage(`failed to get status: ${result.stderr}`);
+        } else {
+            throw new Error(`failed to get status: ${result.stderr}`);
+        }
+    } catch (err) {
+        if (notification) {
+            vscode.window.showErrorMessage(`minikube status failed: ${err}`);
```

Comment:
Again start with capital letter.  How does `Minikube status failed` differ from `Failed to get status` on line 159?  Seems like we should use a consistent form of wording, unless we are intentionally keeping them different e.g. for diagnostic purposes.